packages/protocol/test-sol/devchain/migration/Migration.t.sol

```diff
@@ -1,19 +1,29 @@
 // SPDX-License-Identifier: UNLICENSED
-pragma solidity >=0.8.7 <0.8.20;
+pragma solidity >=0.5.13 <0.8.20;
```

Comment:
I think it will eventually hurt us to have the Foundry tests in 0.5, with a quick glance this doesn't need it.